### PR TITLE
{vis}[GCCcore/9.3.0] X11 v20200222

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/b7d43e2742f8e4d533213b457fa5bb07 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/a38e8eb85a8014f28aabc650aff42b88 for a full test report.

**Comment by akesandgren** [comment]
I think --inject-checksums made a mess of X11 this time...

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/cc5174261c93cc9ee61d85d445ae98cd for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/f9d78921be3f64de2af361d872a467f6 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**FAILED**
Build succeeded for 2 out of 3 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/00ba5696c4367a86227d9ede2019fa79 for a full test report.

File: easybuild/easyconfigs/x/X11/X11-20200222-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    }),
+    ('xtrans', '1.4.0', {  # 2019-03-16
+        'checksums': ['48ed850ce772fef1b44ca23639b0a57e38884045ed2cbb18ab137ef33ec713f9'],
+    }),
+    ('libxkbcommon', '0.10.0', {  # 2020-01-18
+        'easyblock': 'MesonNinja',
+        'sources': ['libxkbcommon-%(version)s.tar.xz'],
+        'checksums': ['57c3630cdc38fb4734cd57fa349e92244f5ae3862813e533cedbd86721a0b6f2'],
+        'preconfigopts': '',
+        'configopts': '-Denable-wayland=false ',
```
Comment (ocaisa) [inline easybuild/easyconfigs/x/X11/X11-20200222-GCCcore-9.3.0.eb]: You either need to introduce a doxygen dep or add `-Denable-docs=false` to the `configopts`

Comment (akesandgren) [reply easybuild/easyconfigs/x/X11/X11-20200222-GCCcore-9.3.0.eb]: Updated

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/652a5cdf4b367211c6299d1dc0664f8e for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/08b7cfa917b5c37da5e6a3ce72ef038b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1fdd463fd127ee34691d430faaf47439 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/5f5b14167fdce8dc5fbbc13f5c6d6701 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/604ad2c90698bcb78ace661b1bdf8979 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bb2c1a09.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/335d93d1836d621479428ea6d7473265 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/27c9288fddb1e51ada5705c50a6c1d73 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c2a810cfe58942c588eb481c80eeb4bb for a full test report.

**Pull request merged**